csunplugged/docker-development-entrypoint.sh

```diff
@@ -5,7 +5,7 @@ function postgres_ready(){
 import sys
 import psycopg2
 try:
-    conn = psycopg2.connect(dbname="postgres", user="postgres", host="localhost")
+    conn = psycopg2.connect(dbname="postgres", user="postgres", host="localhost", port="5434")
```

Comment:
why did this change?

Reply:
See https://github.com/uccser/cs-unplugged/pull/593